certbot-deploy-hook

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+backup() {
+    if [ -e "$1" ]; then
+        # If the user is setting up our automatic certbot-management on a
+        # system that already has certs for Zulip, use some extra caution
+        # to keep the old certs available.
+        mv -f --backup=numbered "$1" "$1".setup-certbot || true
```

Comment:
Maybe just call it `.certbot-bak`?  I think that makes more clear it's a backup file.